torch/_dynamo/functional_export.py

```diff
@@ -105,6 +105,19 @@ def clean_nn_module_stack(
 
         node.meta["nn_module_stack"] = cleaned_stack
 
+    if "dynamo_flat_name_to_original_fqn" in graph_module.meta:
```

Comment:
I feel we shouldn't support legacy behaviours that makes strict-export more complex than whats really needed, but i feel this is better than ppl using another private API to do this.